eventschedules/serializers.py

```
@@ -0,0 +1,16 @@
+
+from .models import EventSchedule
+
+
+class EventScheduleSerializer(serializers.ModelSerializer):
+    """Event schedule serializer."""
+
+    class Meta:
+        """metaclass."""
+
+        model = EventSchedule
+        exclude = ["created_at", "updated_at"]
```

Ruff (0.16.0)

[warning] Mutable default value for class attribute

(RUF012)